<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCFirmwareUpgrade</class>
 <widget class="QWidget" name="SCFirmwareUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>固件下载</string>
   </property>
   <widget class="QToolButton" name="tbnOpenFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFileAddress">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>19</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>固件大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下载日期：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFirmwareSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDateTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123456</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>固件下载</string>
   </property>
   <widget class="QLabel" name="leProgramState">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>运行状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Boot版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="leBoot">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>固件版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="leFirmware">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>硬件版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="leHardware">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnReadFirmwareInfo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读取固件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnDownload">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>固件下载</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnEnterAPP">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>进入APP</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbnReturnBootLoader">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回Boot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
